### ISSUE TITLE
hardcoded color in output

### ISSUE THREAD

**Issue by timhemel** [semgrep/semgrep#4377]
**Describe the bug**
If I run semgrep with a ruleset, part of the output is colored "black" (for some value of black), which happens to be the same color as my terminal background (SolarizedDark color scheme). 

**To Reproduce**
Steps to reproduce the behavior, ideally a link to https://semgrep.dev:
Install a Solarized Dark color scheme in your terminal and run `semgrep -f xyz.yaml`

**Expected behavior**
I expect output to use the current foreground color.

**Screenshots**

The responsible piece of code is in `semgrep/semgrep/formatter/text.py`:

``` python
        line = (
            line[:start_color]
            + with_color(
                "bright_black", line[start_color : end_color + 1]
            )  # want the color to include the end_col
            + line[end_color + 1 :]
        )
```

I am afraid the only proper way to solve this is to implement color schemes in semgrep.

**What is the priority of the bug to you?**

- [ ] P0: blocking your adoption of Semgrep or workflow
- [ ] P1: important to fix or quite annoying
- [x] P2: regular bug that should get fixed

**Environment**
If not using semgrep.dev: are you running off docker, an official binary, a local build?
Local build.

**Comment by ievans** [comment]
@adamboros are you still working on this?

**Comment by adamboros** [comment]
@ievans Yes, I want to push my changes this week

**Comment by ievans** [comment]
Fixed in #4541 :)
